docs/webrtc-implementation.md

```
@@ -154,6 +154,77 @@ function teardownAllConnections() {
+
+```bash
+bun run --cwd apps/server dev
+```
+
+`apps/server/certs/` に `cert.pem` と `key.pem` があれば **https の 8443** で、無ければ http の 3000 で待ち受けます([`apps/server/src/index.ts`](../apps/server/src/index.ts) の `hasTls`)。起動ログの `Hono server listening on ...` で確かめてください。証明書が無ければ `bun run --cwd apps/server cert` で作れます。
+
+#### 2. dev サーバの繋ぎ先は `VITE_HONO_WS_URL` にする
+
+`apps/web/.env.local` を作ります(`*.local` は [`apps/web/.gitignore`](../apps/web/.gitignore) の対象なのでコミットされません)。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)